rocminfo.cc

```diff
@@ -1185,6 +1185,70 @@ AcquireAndDisplayAgentInfo(hsa_agent_t agent, void* data) {
   return HSA_STATUS_SUCCESS;
 }
 
+static hsa_status_t
+showCPUInfo(hsa_agent_t agent, void* data) {
+  int pool_number = 0;
+  int isa_number = 0;
+
+  hsa_status_t err;
+  agent_info_t agent_i;
+
+  int *agent_number = reinterpret_cast<int*>(data);
```

Comment:
It seems agent_number is not used, and it can be removed

Reply:
Currently it is not used, I had thought to use it when we need to index the individual agent display. So removed and updated in the fork branch as provided below. Also made some changes in help message display
https://github.com/ajitclarencejoseph/rocminfo/commit/026828a8a1e968c6121d81394fc4310d2f4757f8